pkgs/development/tools/build-managers/gnumake/4.2/default.nix

```diff
@@ -27,7 +27,8 @@ stdenv.mkDerivation {
   nativeBuildInputs = stdenv.lib.optionals guileSupport [ pkgconfig ];
   buildInputs = stdenv.lib.optionals guileSupport [ guile ];
 
-  configureFlags = stdenv.lib.optional guileSupport "--with-guile";
+  configureFlags = stdenv.lib.optional guileSupport "--with-guile"
+    ++ stdenv.lib.optional stdenv.isDarwin "ac_cv_struct_st_mtim_nsec=no";
```

Comment:
Should this maybe carry a comment, when it should safe to re-enable this feature again?
i.e. a while after `10.13`?